frontend-react/src/components/SideNav.tsx

```diff
@@ -0,0 +1,57 @@
+import { SideNav } from "@trussworks/react-uswds";
+import { useLocation } from "react-router-dom";
+
+import { USNavLink } from "./USLink";
+
+export interface SideNavItemProps
+    extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+    items?: React.ReactNode[];
+    children: React.ReactNode;
+    to?:
+        | string
+        | React.FunctionComponent<
+              React.AnchorHTMLAttributes<HTMLAnchorElement> & {
+                  children: React.ReactNode;
+              }
+          >;
+    isActive?: boolean;
+}
+
+/**
+ * Wrapper component that provides the NavLink and SideNav sibling elements (if items
+ * provided). The parent navlink can be customized by passing a functional component
+ * or html element string as the "to" prop (defaults to USNavLink). The SideNav sibling
+ * can be forced visible via the "isActive" prop (otherwise defaults to checking path).
+ */
+export function SideNavItem({
```

Comment:
Can we `export default` here?  I'm thinking there should always be a default export in component files to encourage us to have a 1:1 between files and components.  There'll be times that we'll have "sub-components" of course, but that's something we should try to avoid whenever possible.